### PR TITLE
Move EntityIdComposer from Wikibase to DataModelServices

### PR DESCRIPTION
**Pull request by Ladsgroup**

### CODE REVIEW

File: src/EntityId/EntityIdComposer.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+use Wikibase\DataModel\Entity\EntityId;
+
+/**
+ * Constructs EntityId objects from entity type identifiers and unique parts of entity ID
+ * serializations. The unique part is typically the numeric part of an entity ID, excluding the
+ * static part that's the same for all IDs of that type.
+ *
+ * Meant to be the counterpart for @see Int32EntityId::getNumericId, as well as an extensible
+ * replacement for @see LegacyIdInterpreter::newIdFromTypeAndNumber.
+ *
```
Comment (thiemowmde) [inline src/EntityId/EntityIdComposer.php:16]: Can you add a `@since` tag?

File: tests/unit/EntityId/EntityIdComposerTest.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+use PHPUnit_Framework_TestCase;
+use UnexpectedValueException;
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Services\EntityId\EntityIdComposer;
+
+/**
+ * @covers Wikibase\DataModel\Services\EntityId\EntityIdComposer
+ *
+ * @group Wikibase
```
Comment (thiemowmde) [inline tests/unit/EntityId/EntityIdComposerTest.php]: Please remove, I think we don't use `@group` tags in this component.

**Review by thiemowmde — CHANGES_REQUESTED** [review]
It even said "please move me", so yay! Just a few documentation details.

File: tests/unit/EntityId/EntityIdComposerTest.php (hunk 2 above)
Comment (Ladsgroup) [reply tests/unit/EntityId/EntityIdComposerTest.php]: Done

File: src/EntityId/EntityIdComposer.php (hunk 1 above)
Comment (Ladsgroup) [reply src/EntityId/EntityIdComposer.php:16]: Done

**Pull request merged**